src/format.js

```diff
@@ -524,6 +524,91 @@ const formatMetadatas = metadatas => {
   }))
 }
 
+const formatReceiptStatements = receipts => {
+  let balanceChangeReceipt = []
+  let balanceTransferReceipt = []
+  let inflationReceipt = []
+  let artifactExpiryReceipt = []
+
+  receipts.map(receipt => {
```

Comment:
Since there is no assignment, I think it's can be better with forEach()

Reply:
sure, but what is mean no assignment?